onnx_tf/doc/support_status.md

```diff
@@ -0,0 +1,233 @@
+ONNX-Tensorflow Support Status
+======
+
+Backend
+______
+
+| ONNX Op        | Supported Version  |
+| -------------- |:------------------:|
+|unsqueeze|1|
+|arg_min|1|
+|less|1|
+|lp_normalization|1|
+|shape|1, 2, 3, 4, 5|
```

Comment:
Not relate to this pr, does `1, 2, 3, 4, 5` make user a little confused?

Reply:
true. Maybe I'll fix the way trivial ops are handled.